trlx/sweep.py

```diff
@@ -89,9 +323,39 @@ def tune_function(train_function, param_space: dict, tune_config: dict, resource
     # convert a nested path to a module path
     script_path = args.script.replace(".py", "").replace("/", ".")
     script = importlib.import_module(script_path)
-    # Register the training function that will be used for training the model.
-    tune.register_trainable("train_function", script.main)
-    tune_function(script.main, param_space, tune_config, resources)
+    project_name = "sweep_" + script_path.split(".")[-1]
+
+    default_config["train"]["project_name"] = project_name
+    default_config["train"]["group_name"] = datetime.now().replace(microsecond=0).isoformat()
+    param_space["default_config"] = default_config.copy()
+    param_space["default_config"]["train"]["git_tag"] = get_git_tag()
+    param_space_train = {"train_loop_config": param_space}
+
+    tuner = tune.Tuner(
+        AccelerateTrainer(
+            script.main,
+            # Mandatory arg. None means use Accelerate default path
+            accelerate_config_path=args.accelerate_config,
+            scaling_config=ScalingConfig(
+                trainer_resources={"CPU": 0},
+                num_workers=args.num_gpus,
+                use_gpu=True,
+                resources_per_worker={"CPU": args.num_cpus},
```

Comment:
just to be a bit more explicit
```suggestion
                resources_per_worker={"CPU": args.num_cpus, "GPU": 1},
```